content/_prerequisites/02-environment-setup/03-docker.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Docker"
+---
+
+Using VS Code's extension [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) to create Presidium content is a simple and consistent way to set up a development environment without needing to update any configuration on your local machine.
+
+## Required Software
+**macOS:** 
+- [Docker Desktop](https://www.docker.com/products/docker-desktop) 2.0+
+- [VS Code](https://code.visualstudio.com/) or [VS Code Insiders](https://code.visualstudio.com/insiders/)
+
+**Linux:** 
+- [Docker CE/EE](https://docs.docker.com/install/#supported-platforms) 18.06+ 
+- [Docker Compose](https://docs.docker.com/compose/install) 1.21+. (The Ubuntu snap package is not supported.)
+- [VS Code](https://code.visualstudio.com/) or [VS Code Insiders](https://code.visualstudio.com/insiders/)
+
+**Windows:** 
+- [Docker Desktop](https://www.docker.com/products/docker-desktop) 2.2+
+- [WSL2 back-end](https://aka.ms/vscode-remote/containers/docker-wsl2). (Docker Toolbox is not supported.)
+- [VS Code](https://code.visualstudio.com/) or [VS Code Insiders](https://code.visualstudio.com/insiders/)
+
+## Installation
+
+1. Install any required software (above).
+
+2. Ensure that git user config is configured on your **local** environment: 
+    ```bash
+    git config --global user.name "Your Name"
+    git config --global user.email "your.email@address"
+    ```
+    **Note:** If a Git Credential Manager (e.g. Github Desktop) is configured on the local environment, this step can be skipped.
+
+
+4. Inside VS Code install the extension: [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers).
+
+## Setting up your project
+1. Copy the link to the Github repository to be cloned.
+
+1. In [VS Code](https://code.visualstudio.com/) open the command pallet (F1 or &#8984;&#8679;`P`) and type 
+`>Remote-Containers: Clone Repository in Container Volume...`
+    
+    **Note**: This step replaces the first step of the [Getting Started](/getting-started/#download-the-template) instructions.
+
+1. Paste the URL for the repository which should be cloned.
+
+1. A popup asking which branch to clone may appear. Select the one you want to checkout initally, once the container is created you are free to checkout any branch in the repository.
+
+1. When prompted for a [Volume](https://docs.docker.com/storage/volumes/) to use there are a few options:
+    - (Recommended) Create a single volume for all project. This is the most space efficient, and unless there is a reason two project should not share a filesystem, there is no harm to this for most development environments.
```

Comment:
Create a single volume for all projects

Reply:
and unless there is a reason two projects

Reply:
there is no harm **in** this

Reply:
```suggestion
    - (Recommended) Create a single volume for all projects. This is the most space efficient, and unless there is a reason two projects should not share a filesystem, there is no harm in this for most development environments.
```